src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,57 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+<tags:master pageTitle="Cart">
+    <p>
+        <c:choose>
+            <c:when test="${not empty errorMap}">
+                <span class="message-red">Error adding to cart!</span>
+            </c:when>
+            <c:when test="${param.success}">
+                <span class="message-green">Added to cart successfully!</span>
+            </c:when>
+        </c:choose>
+    </p>
+    <form method="post" action="${pageContext.servletContext.contextPath}/cart">
+        <table>
+            <tr>
+                <td>Image</td>
+                <td>Description</td>
+                <td>Price</td>
+                <td>Quantity</td>
+                <td>Action</td>
+            </tr>
+            <c:forEach var="item" items="${cart.cartItems}" varStatus="status">
```

Comment:
If the cart is empty we will get empty table with headers - consider checking for epmty table before rendering the table.